HW_module_2.py

```diff
@@ -0,0 +1,54 @@
+import string
+import random
+
+# create a list of random number of dicts (from 2 to 10)
+
+dict_List = []                                             # empty list
+for x in range(random.randint(2, 10)):                     # random number of dictionaries
+#    size = random.randint(1,26)                          # random dictionary size
+    size = 3                                               # the size of dictionary
+    keys = random.sample(string.ascii_lowercase, size)     # random lowercase letters in key parameters
+    values = (random.randint(1, 50) for y in range(size))  # random numbers in value parameters
```

Comment:
Why do you need the generator here?  values = random.sample(range(1, 100), size)
In some cases, a generator can be a better option, but I'm not sure that it was used here intentionally.